src/styles/stopwatch.scss

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  height: $stopwatchRadius;
+  width: $stopwatchRadius;
+  border-radius: $circle;
+  border: 1vmin dotted black;
```

Comment:
Consider using a variable for the `1vmin` value in the `border` property to maintain consistency and scalability. If you decide to change the border thickness later, having it as a variable would make it easier to update.